Viewer/src/configuration/configuration.ts

```diff
@@ -94,8 +97,10 @@ export interface ViewerConfiguration {
             specular?: { r: number, g: number, b: number };
         }
         hideLoadingDelay?: number;
+        /** Deprecated */

```

Comment:
Should we keep them if not use ?

Reply:
yes, just for d.ts reasons (I dont want to break public projects'build process)